optimum/intel/openvino/modeling_diffusion.py

```diff
@@ -95,6 +95,13 @@
     FluxPipeline = StableDiffusionPipeline
 
 
+if is_diffusers_version(">=", "0.31.0"):
+    from diffusers import FluxImg2ImgPipeline, FluxInpaintPipeline
+else:
+    FluxImg2ImgPipeline = StableDiffusionImg2ImgPipeline
+    FluxInpaintPipeline = StableDiffusionInpaintPipeline
```

Comment:
With this, will ov flux pipeline still work on `diffusers<0.31`?

Reply:
I see. Can this be done then? 
```
    FluxImg2ImgPipeline = object
    FluxInpaintPipeline = object
```

If so, I think this is a better option because it is more clear that in such case we don't expect it to work/be instantiated.

Reply:
Or even
```
    FluxImg2ImgPipeline = None
    FluxInpaintPipeline = None
```

Reply:
applied